### ISSUE TITLE
How to use min_detection_confidence in IOS code

### ISSUE THREAD

**Issue by LQ921003** [google/mediapipe#2376]
I want to set this value.But i don't know where to set.
I use **Holistic** track.
How to set min_detection_confidence？
Where to set min_detection_confidence？

**Comment by WIndFate** [comment]
I'm also want to know..., please help us

**Comment by sgowroji** [comment]
Hi @LQ921003,  You can do the changes in the graph https://github.com/google/mediapipe/blob/b544a314b3c4818a4d6c56964b020c5be945377a/mediapipe/modules/pose_detection/pose_detection_gpu.pbtxt#:~:text=%7D-,%23%20Decodes%20the%20detection%20tensors%20generated%20by%20the%20TensorFlow%20Lite%20model,%7D,-%23%20Performs%20non-max at min_score_thresh: 0.5

**Comment by LQ921003** [comment]
Hi @sgowroji ,Thank you very much for your reply.
I tried it just now and it seems to work.But I encountered a similar problem. When there is no one in the screen, there will still be lines of recognition.I set the value of min_score_thresh to 0.8.This problem is manifested in this picture file.
![IMG_0195](https://user-images.githubusercontent.com/82488102/129149432-b062b957-11b7-4805-a1ff-3f22b83cef33.jpg)
